### ISSUE TITLE
underline focus/hover states for link text in main page content

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/rockarch.org#48]
- Links in main page content should have underline focus/hover states to better conform to WCAG 2.1 success criteria 1.4.1 (see https://www.w3.org/WAI/WCAG21/Techniques/general/G183).
